tmt/steps/provision/testcloud.py

```diff
@@ -515,6 +534,10 @@ def start(self):
                 self._instance.pci_net = "virtio-net-pci"
 
         # Prepare ssh key
+        # TODO: Maybe... some better way to do this?
```

Comment:
+1 to expose to the upper levels. We could make it part of hardware (so it can be used for other provision plugins).
It does feel a bit off topic for that field though (hardware is about what plan _needs_, this one is more about what image used really _is_.)

@psss @happz @thrix

Reply:
Yeah, this sounds useful. But I guess we do not have to block this PR on that.